tox.ini

```diff
@@ -1189,6 +1189,17 @@ changedir = docs
 commands =
   sphinx-build -E -a -W -b html -T . _build/html
 
+[testenv:lint]
+basepython: python3
+recreate = True
+deps =
+  -r dev-requirements.txt
+
+commands =
+  black --config {toxinidir}/pyproject.toml {{toxinidir}} --diff --check
+  isort --settings-path {toxinidir}/.isort.cfg {{toxinidir}} --diff --check-only
+  flake8 --config {toxinidir}/.flake8 {toxinidir}
```

Comment:
It's also a bit confusing to have a `lint` test environment that does not run `pylint` like other test environments do...

Reply:
If it wasn't a pain to run pylint for a multi package repository I would gladly add it :sweat_smile: